### ISSUE TITLE
java.lang.IllegalArgumentException: Cannot set property PropertyDirection on block randomthings:plate_extraction

### ISSUE THREAD

**Issue by Pythonic-Rainbow** [lumien231/Random-Things#329]
https://pastebin.com/zWLFxeZ6

**Comment by tyler489** [comment]
In short: This fails:

```
((ItemBlock) itemStack.getItem()).getBlock().getStateFromMeta(itemStack.getItem().getMetadata(itemStack.getMetadata()));
```
It looks very convoluted, but this is how vanilla's ItemBlock+Block work together to convert an item damage to a blockstate.

**Comment by brisingraerowing** [comment]
Appears to be a duplicate of the already fixed #326

**Comment by lumien231** [comment]
It is
